package.json

```diff
@@ -1232,6 +1232,109 @@
                         ]
                     }
                 }
+            },
+            {
+                "type": "docker-compose",
+                "properties": {
+                    "dockerCompose": {
```

Comment:
I kind of like the use of nested properties to determine the sub-command for the task (i.e. `up` vs. `down`).  I may steal that for other extensions.  That might have also worked for our Docker `build` and `run` tasks, but I suppose it's too late to for that.  :-)

Reply:
Yeah. I was thinking to myself--`docker-compose down` is barely complicated enough to justify a task of its own, how can I do this in the laziest way possible? :smile: